dialect_postgresql.go

```diff
@@ -47,6 +47,19 @@ func (p *postgresql) Details() *ConnectionDetails {
 	return p.ConnectionDetails
 }
 
+func (p *postgresql) Quote(key string) string {
+	parts := strings.Split(key, ".")
+
+	for i, part := range parts {
+		part = strings.Trim(part, `"`)
+		part = strings.TrimSpace(part)
+
+		parts[i] = p.commonDialect.Quote(part)
+	}
+
+	return strings.Join(parts, ".")
+}
```

Comment:
How about moving this to commonDialect? Then it can be simplified to:
- if key has only one part (so no dot in key) => return key surrounded by quotes.
- otherwise, split key on the dots, call Quote on each part, then join the result.